draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -880,7 +880,60 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 # IANA Considerations {#iana_considerations}
 
-TBD
+## Media Type Registration
+
+This section registers the "application/sdjwt-issuance" media type [@RFC2046] in
```

Comment:
I hate to bikeshed but SD-JWT is hyphenated everywhere else in this draft so `sdjwt-issuance` and `sdjwt-presentation` look a bit off.  Is there a reason not to go w/ `sd-jwt-issuance` and `sd-jwt-presentation`?

Reply:
or if length is of concern, something like `sd-jwt-i` and `sd-jwt-p`

Reply:
Please bikeshed :)

```suggestion
This section registers the "application/sd-jwt-issuance" media type [@RFC2046] in
```